src/chapters/git/exercises.rst

```diff
@@ -134,6 +134,72 @@ a different service to create and share remote repositories.
 If you will be using GitHub, create your account by following these steps:
 
 #. Navigate to GitHub's site using the link above.
-#. Sign up for an account on the homepage either by filling out the form or
-   clicking the *Sign Up* button.
-#. Once you have an account, you are ready to store your remote work!
+#. Click the *Sign Up* button and follow the on-screen directions.
+#. Create a *Personal Access Token* (see below).
+#. Once you have an account and a personal access token, you are ready to store
+   your remote work!
+
+Create a Personal Access Token (PAT)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. index:: personal access token
+
+To push and pull from GitHub, users must create a **personal access token**. A
+PAT takes the place of a password, and the token process is considered more
+secure than a username/password verification.
+
+After you create your PAT, you will use it anytime you interact with GitHub
+from the terminal.
+
+.. admonition:: Example
+
+   In the terminal, interactions between your computer and GitHub might look
+   something like this:
+
+   .. sourcecode:: bash
+
+      $ git push origin main
+      Username: your_username
+      Password: your_token
+
+   The bottom two lines will prompt you to enter your GitHub username and
+   personal access token.
+
+Some users question the need for a PAT, since it looks like another password
+they have to remember. Rather than diving into a long debate and explanation,
+we'll focus on the main point: *GitHub requires a PAT or similar token*. The
+service is incredibly helpful, and we want to use it, so we'll follow the
+developers' advice.
+
+Creating a PAT for your account is fairly straightforward. GitHub provides
+detailed instructions, so we will use their words. Login to your account, then
+carefully follow steps 1 - 9 for
+`Creating a Token <https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token#creating-a-token>`__.
+
```

Comment:
I like this video that Courtney used in the C# book: https://www.youtube.com/watch?v=kHkQnuYzwoo, but he does cache the user name and PAT

Reply:
That video is great!  It's very clear.

However, it does go deeper into the git config commands than is necessary for the HS students.  As you noted in your other comment, the students will be using shared machines.  They really shouldn't be be storing their usernames and tokens indefinitely in the cache.

I can add the link with a time parameter (e.g. "Watch up until 4:13"), but the kids can easily go past that.  What do you think?

Reply:
I did ask Dan if they use a password saving app and his school doesn't, I have had schools that do.  If they are doing local development on a shared computer, where is their local file of code stored?  I agree that the way it is written now for 60 mins is the ideal.  And just like we get use to the git status, git add . git commit -m"", and git push, they could get use to signing in to their git hub account and getting a new PAT every class and following the steps in the first five minutes?

My thoughts are we should merge, and use the teachers as test subjects tomorrow and then create a new PR to address any changes need?

Reply:
I agree.  Thanks.